src/products/android-l10n/removing_projects.md

```diff
@@ -1,22 +1,22 @@
 # Removing projects from android-l10n
 
-Below are listed the steps to follow in order to safely remove projects from [android-l10n](https://github.com/mozilla-l10n/android-l10n).
+Follow the steps below to safely remove projects from [android-l10n](https://github.com/mozilla-l10n/android-l10n).
 
 ## Contact Release Engineering
 
-The first thing to do is to request from the Release Engineering team (“RelEng”) that they disable automation associated to the [android-l10n project](https://github.com/mozilla-l10n/android-l10n) you want to remove.
-Currently, the way to make this request is to send an email directly to RelEng (“Aki” and “Mihai”).
+First, make a request to the Release Engineering team (“RelEng”) to disable the automation associated with the [android-l10n project](https://github.com/mozilla-l10n/android-l10n) you want to remove.
+Currently, the way to make this request is to send an email directly to RelEng at releng(at)mozilla(dot)com.
 This step will help ensure that any automation linked to this product will be discontinued, as it will remove all config/automation/localizable files and prevent future updates to release.
 
 ## RelEng files a PR
 
 RelEng will then create a PR in the [android-l10n-tooling GitHub repository](https://github.com/mozilla-l10n/android-l10n-tooling). Make sure to wait for that PR to be merged by RelEng before continuing to the next steps, since otherwise, automation will not have been stopped.
 
-Example PR for the removal of Firefox for Fire TV product: https://github.com/mozilla-l10n/android-l10n-tooling/pull/29
+See [example PR](https://github.com/mozilla-l10n/android-l10n-tooling/pull/29) for the removal of Firefox for Fire TV product.
 
-## Stop sync for the android-l10n project in Pontoon
+## Stop sync for the project in Pontoon
 
-In order to disable sync in Pontoon, log in to [the admin view of android-l10n](https://pontoon.mozilla.org/admin/projects/android-l10n/), scroll to the bottom of the page, and click on “sync disabled” on the left.
+In order to disable sync in Pontoon, log in to admin view of the project (e.g. [Firefox for Android](https://pontoon.mozilla.org/admin/projects/firefox-for-android/), [Focus for Android](https://pontoon.mozilla.org/admin/projects/focus-for-android/), etc.) and scroll to the bottom of the page. Click the “Sync Disabled” checkbox on the left and then "Save Project".
```

Comment:
I would suggest to use the `DISABLED` checkbox, since the goal is to remove the project (we never actually remove projects from Pontoon, only disable them).

Reply:
Thanks, this makes sense. I've adjusted the wording of the whole section to align with this.